cluster/addons/metrics-server/metrics-apiservice.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiregistration.k8s.io/v1beta1
+apiVersion: apiregistration.k8s.io/v1
```

Comment:
LGTM because of https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.10/#apiservice-v1-apiregistration-k8s-io